org.cvfosammmm.Porto.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "org.cvfosammmm.Porto",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "porto",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux",
+        "--env=LC_ALL=C",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+            "name": "gtksourceview",
+            "config-opts": ["--enable-gtk-doc=no"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz",
+                    "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd"
+                }
+            ]
+        },
+        {
+            "name": "pythondependencies",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "python3 -m venv /app",
+                "source /app/bin/activate",
+                "pip3 install nbformat PyGObject pypandoc markdown bleach jupyter jupyter_client"
+            ]
+        },
+        {
+            "name": "porto",
+            "buildsystem": "simple",
+            "build-commands": [
+                "source /app/bin/activate",
+                "mkdir -p /app/share/applications",
+                "cp resources/org.cvfosammmm.Porto.desktop /app/share/applications/org.cvfosammmm.Porto.desktop",
+                "mkdir -p /app/share/icons/hicolor/scalable/apps",
+                "cp resources/images/org.cvfosammmm.Porto.svg /app/share/icons/hicolor/scalable/apps/org.cvfosammmm.Porto.svg",
+                "mkdir -p /app/share/metainfo",
+                "cp resources/org.cvfosammmm.Porto.appdata.xml /app/share/metainfo/org.cvfosammmm.Porto.appdata.xml",
+                "mkdir -p /app/bin",
+                "cp -R app /app/bin/",
+                "cp -R cell /app/bin/",
+                "cp -R dialogs /app/bin/",
+                "cp -R helpers /app/bin/",
+                "cp -R resources /app/bin/",
+                "cp -R result_factory /app/bin/",
+                "cp -R notebook /app/bin/",
+                "cp -R workspace /app/bin/",
+                "install -D __main__.py /app/bin/porto"
```

Comment:
Not a blocker but this should be probably handled by a proper build system upstream.